src/smallmap_gui.cpp

```diff
@@ -478,6 +478,23 @@ static inline uint32 GetSmallMapRoutesPixels(TileIndex tile, TileType t)
 
 		const SmallMapColourScheme *cs = &_heightmap_schemes[_settings_client.gui.smallmap_land_colour];
 		return ApplyMask(cs->default_colour, &andor);
+	} else if (t == MP_ROAD) {
```

Comment:
should be transformed into a switch?

Reply:
Possibly. It does create a quite a bit of extra identation.